server/lib/src/utils/element_helper.dart

```diff
@@ -492,8 +486,10 @@ class ElementHelper {
     Duration? settleBetweenScrollsTimeout,
     Duration? dragDuration,
   }) async {
-    delta ??= defaultScrollDelta;
-    maxScrolls ??= defaultScrollMaxIteration;
+    delta ??= FlutterDriver.instance.settings
+        .getSetting('flutterScrollDelta');;
```

Comment:
We could use the Settings key for fetching the values? 

```dart
FlutterDriver.instance.settings.getSetting(FlutterSettings.flutterScrollDelta)
```

Reply:
Thanks or prompt review @sudharsan-selvaraj,I have incorporated the review comments